### PR TITLE
Fixes margin and layout

### PR DESCRIPTION
**Pull request by kul3r4**
Fixes:

- amp-font
- amp-fx-flying-carpet
- amp-gfycat

### CODE REVIEW

File: src/20_Components/amp-font.html [hunk 1]
```diff
@@ -94,13 +93,13 @@
     on-load-add-class="tangerine-loaded">
   </amp-font>
     <!-- and use an existing font. -->
-  <p class="tangerine">
+  <p class="tangerine p1">
```
Comment (sebastianbenz) [inline src/20_Components/amp-font.html]: Shouldn't these be catched by our generic rule: 
```
.abe-preview p { 
  padding: 1rem;
}
```

File: src/20_Components/amp-fx-flying-carpet.html [hunk 2]
```diff
@@ -100,14 +100,13 @@
 
      The text that follows is to ensure that the `amp-fx-flying-carpet` is placed before the last viewport.
   -->
-  <p>
+  <p class="p1">
```
Comment (sebastianbenz) [inline src/20_Components/amp-fx-flying-carpet.html]: same as above

Comment (kul3r4) [inline src/20_Components/amp-fx-flying-carpet.html]: I added that rule at the component sample level for amp-bind, should I move it to example.css?

Comment (sebastianbenz) [inline src/20_Components/amp-fx-flying-carpet.html]: yes

Comment (kul3r4) [inline src/20_Components/amp-fx-flying-carpet.html]: Just moved it, thanks

**Pull request merged**